resource/docker/network/network.go

```diff
@@ -79,15 +79,15 @@ func (n *Network) Check(context.Context, resource.Renderer) (resource.TaskStatus
 			n.diffNetwork(nw)
 		}
 
-		ok, err := n.isCreatable(nw)
+		invalidGWs, err := n.isCreatable(nw)
 		if err != nil {
 			n.RaiseLevel(resource.StatusFatal)
 			return n, err
 		}
 
-		if !ok {
+		if invalidGWs != nil && len(invalidGWs) > 0 {
 			n.RaiseLevel(resource.StatusCantChange)
-			return n, err
+			return n, fmt.Errorf("%v: gateway(s) are already in use", invalidGWs)
```

Comment:
When there is more than one gateway, readability is an issue. Maybe use some formatting to separate the values.